README.md

```diff
@@ -8,10 +8,31 @@ The stack also creates a small EC2 instance (defined in ec2-test.tf) that will b
 
 ## Deployment
 
+This stack is meant to be consumed as a module in your existing terraform stack. You can consume it by using code similar to this:
+
+    ````
+    module "ekk_stack" {
+        source = "github.com/GSA/devsecops-ekk-stack"
```

Comment:
It would actually be `github.com/GSA/devsecops-ekk-stack//terraform` - confirmed by adding to devsecops-example: https://github.com/GSA/devsecops-example/pull/65. Alternatively, can stick to the [standard module structure](https://www.terraform.io/docs/modules/create.html#standard-module-structure) and put the reusable module in the root directory.

Reply:
Fixed.